src/shared/utils/validateMonthString.ts

```diff
@@ -0,0 +1,3 @@
+const MONTH_VALIDATION_REGEX = /^(0[1-9]|1[0-2])$/;
+
+export const validateMonthString = (month: string) => new RegExp(MONTH_VALIDATION_REGEX).test(month);
```

Comment:
```suggestion
type Month = `0${1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9}` | `1${0 | 1 | 2}`;

export const validateMonthString = (month: string): is Month  => new RegExp(MONTH_VALIDATION_REGEX).test(month);
```

이렇게 하면 타입을 확정지을 수 있어요!

Reply:
> 이렇게 하면 타입을 확정지을 수 있어요!

잘 활용하겠습니다!